app/src/main/java/com/codepath/apps/critterfinder/adapters/PetFavoritesAdapter.java

```diff
@@ -37,6 +37,9 @@ public PetFavoritesAdapter(List<PetModel> petModels) {
         public TextView tvPetFavName;
         public TextView tvPetFavSex;
         public CircleImageView ivPetFavImage;
+        public TextView tvPetFavSize;
+        public TextView tvPetFavAge;
+        public TextView tvPetFavBreeds;
 
```

Comment:
just sharing a tip for future learning. You can use butterknife inside the viewholder too! check out our SwipeableCardAdapter for an example